dev-libs/doctest/doctest-2.4.6.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="The fastest feature-rich C++11/14/17/20 single-header testing framework"
+HOMEPAGE="https://github.com/onqtam/doctest"
+SRC_URI="https://github.com/onqtam/doctest/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND=""
```

Comment:
We can [drop](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3) all 3 of these lines.

Reply:
thanks